docs/notebook_tutorial/ocr.ipynb

```diff
@@ -409,8 +456,9 @@
   }
```

Comment:
Let's have a better summary here. For example, we can describe what Forte just do, why it can help you, and what you can try on your own etc.

Reply:
added